build-scripts/cassandra-artifacts.sh

```diff
@@ -50,6 +50,12 @@ for x in $(seq 1 3); do
     ant clean artifacts
     RETURN="$?"
     if [ "${RETURN}" -eq "0" ]; then
+        ant -Ddependency-check.home=/tmp/dependency-check dependency-check
+        RETURN="$?"
+        if [ ! "${RETURN}" -eq "0" ]; then
+          echo "Failing the build on OWASP dependency check. Run 'ant dependency-check' locally and consult build/dependency-check-report.html to see the details."
+          break
+        fi
```

Comment:
is the check faster than `ant eclipse-warnings` ? (otherwise it should go afterwards, for the sake of fail as-fast-as-possible)